### ISSUE TITLE
the "db_eval_iou" function of  "jaccard.py" 

### ISSUE THREAD

**Issue by InstantWindy** [fperazzi/davis-2017#16]
I'm sorry, I'm not clear about the input params of the "db_eval_iou" function.Is the input parameter as follows? Thanks!
![image](https://user-images.githubusercontent.com/23608180/49066130-21558a80-f25b-11e8-91b4-841c08feba75.png)

**Comment by InstantWindy** [comment]
Is the value of annotation and segmentation 0 and 1? Is the threshold of segmentaion binarization 0.5? Or different models have different thresholds.

**Comment by fperazzi** [comment]
Yes, either 0 (background) or 1 (foreground). The threshold to binarize the
segmentation is arbitrary. But 0.5 is most likely what is often used.

**Comment by InstantWindy** [comment]
Yeah,I got it ,thank you very much . Is the input parameter of "db_eval_boundary" function of the "f_boundary.py" the same as the input parameter of the "db_eval_iou" function?

**Comment by InstantWindy** [comment]
Thank you ,when I run the "f_boundary.py " code ,it appears error as follows
![image](https://user-images.githubusercontent.com/23608180/49085328-c177d780-f28c-11e8-9770-87afe0e699cb.png)

![image](https://user-images.githubusercontent.com/23608180/49085480-14518f00-f28d-11e8-8c27-5914bd76a0eb.png)
![image](https://user-images.githubusercontent.com/23608180/49085521-2a5f4f80-f28d-11e8-8788-39fe5e55841d.png)

**Comment by fperazzi** [comment]
I think it's a problem with the shape of your input. Look up for the
documentation of the morphology (binary dilation) in `scikit-image`

**Comment by sydney0zq** [comment]
@fperazzi  Hi, I notice that the `jaccard` can only take binary mask (0-1) to calcuate the J. While the `f_boundary` can also take multi-instances (0~k) to calcuate the F measure. Is it right?

Thank you.
